server/selinux/server/pulp-celery.te

```diff
@@ -59,6 +60,8 @@ allow celery_t pulp_var_run_t:file manage_file_perms;
 allow celery_t pulp_var_run_t:dir manage_dir_perms;
 files_pid_filetrans(celery_t, pulp_var_run_t, file)
 
+# needed for ostree
+allow celery_t gpg_exec_t:file { read execute open execute_no_trans };
```

Comment:
Is there a refpol statement that is equivalent to this? Usually I ask in #selinux on freenode or on tech-list. It would make it look more [line #55](https://github.com/beav/pulp/blob/e677845b69189309f570b6a3a0650536c6e91d92/server/selinux/server/pulp-celery.te#L55).

It would be more like allow:

```
celery_t gpg_exec_t:file exec;
```

Or maybe a macro to have celery_t receive the execute like [line #98](https://github.com/beav/pulp/blob/e677845b69189309f570b6a3a0650536c6e91d92/server/selinux/server/pulp-celery.te#L98)